styles/gallery.css

```diff
@@ -0,0 +1,41 @@
+body {
+    background-color: rgb(102, 192, 171);
+    } 
+    
+    h1 {
```

Comment:
These rulesets are indented too far in. They should be at the same level as body on line 1.